README.md

```diff
@@ -197,6 +198,16 @@ class { 'sensu':
 }
 ```
 
+### Enterprise Support
+
+In order to activate enterprise support the license file needs to be added:
+
+```puppet
+class { 'sensu::backend':
+  license_source => 'puppet:///modules/site_sensu/license.json',
```

Comment:
This is the type of code you would have in a profile, so perhaps

`'puppet:///modules/profile/sensu/license.json'`

Reply:
Also updated other places where referenced `site_sensu` and replaced with `profile/sensu`.